tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest referring back to Figure 2a, which I assume shows this aoi geometry.  You might also want to include an interactive folium map showing the aoi, since you use folium later in the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done - opted for figure reference.</p>